Makefile

```diff
@@ -91,13 +90,13 @@ build-images:
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 .PHONY: deploy
 deploy: generate-operator-yaml
-	kubectl apply -f deploy/operator.yaml -n $(CONTROLLER_NAMESPACE)
-	kubectl apply -f deploy/crds/policy.open-cluster-management.io_configurationpolicies.yaml -n $(CONTROLLER_NAMESPACE)
-	kubectl set env deployment/$(IMG) -n $(CONTROLLER_NAMESPACE) WATCH_NAMESPACE=$(WATCH_NAMESPACE)
+	kubectl apply -f deploy/operator.yaml -n $(KIND_NAMESPACE) 
+	kubectl apply -f deploy/crds/policy.open-cluster-management.io_configurationpolicies.yaml -n $(KIND_NAMESPACE)
+	kubectl set env deployment/$(IMG) -n $(KIND_NAMESPACE) WATCH_NAMESPACE=$(WATCH_NAMESPACE)
```

Comment:
This is going to break the instructions in the README, which is fine, but would you please take out (or update) the README to match? 
https://github.com/open-cluster-management-io/config-policy-controller/blob/377109a73a5e9c555e4f45ffc461ba768215abf2/README.md?plain=1#L123-L153

Reply:
Great idea!!